MySQL/01 MySQL - Introduction.md

```diff
@@ -117,13 +139,7 @@ It's good practice to write your commands and keywords in upper-case. It's not r
 
 [go to top](#top)
 
-It's good practice to draw out your intended data input to see how you can design the schemas and tables.
-
-![Student Table](E:\coding-notes\MySQL\assets/Student Table.PNG)
```

Comment:
Instead of deleting the image completely, can change the link to relative instead? The image files are there https://github.com/methylDragon/coding-notes/tree/master/MySQL/assets haha..